acoustid/data/meta.py

```
@@ -62,16 +64,50 @@ def generate_meta_gid(values: dict[str, Any]) -> uuid.UUID:
+                "Inserted meta %d with gid %s and values %r",
+                meta_id,
+                meta_gid,
+                values,
+            )
+            return meta_id, meta_gid
+
+        time.sleep(0.05 * retry)
+
+        retry += 1
+        if retry == max_retries:
+            raise RuntimeError("Failed to insert meta")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)